docs/extending/index.md

```diff
@@ -5,14 +5,8 @@ has_children: true
 
 # Understand and Extend the Kiota generator
 
-For an overview of how the code generation process works, see the [Design Overview](designoverview).
+For an overview of how the code generation process works, see the [design overview](../designoverview.md).
 
 ## Project Structure
 
-For more information on the various components used by the Kiota SDK, see the following topics:
-
-- [Request Builders](requestbuilders)
-- [Models](models)
-- [Abstractions](kiotaabstractions)
-- [Core Library](corelibrary)
-- [Authentication](authentication)
+For more information on the various components used by a Kiota-generated SDK, see the following topics.
```

Comment:
I'm wondering whether we should keep that sentence as we don't have links anymore below?

Reply:
They're there in the rendered content. Basically the Jekyll template used creates a "table of contents" below the rest of the content when there are child topics. See https://jasonjoh.github.io/kiota/extending/

Reply:
thanks for the clarification!